mod_reforged/hooks/config/spawnlist_master.nut

```diff
@@ -1,5 +1,133 @@
+::logInfo("spawnlist_master")
+
+::Const.World.Spawn.Troops.Wardog.Strength = 6;
+::Const.World.Spawn.Troops.Wardog.Cost = 6;
+::Const.World.Spawn.Troops.BanditThug.Strength = 12;
+::Const.World.Spawn.Troops.BanditThug.Cost = 12;
+::Const.World.Spawn.Troops.BanditMarksmanLOW.Strength = 12;
+::Const.World.Spawn.Troops.BanditMarksmanLOW.Cost = 12;
+::Const.World.Spawn.Troops.BanditMarksman.Strength = 24;
+::Const.World.Spawn.Troops.BanditMarksman.Cost = 24;
+::Const.World.Spawn.Troops.BanditRaiderWolf.Strength = 30;
+::Const.World.Spawn.Troops.BanditRaiderWolf.Cost = 30;
+::Const.World.Spawn.Troops.BanditRaider.Strength = 30;
+::Const.World.Spawn.Troops.BanditRaider.Cost = 30;
 ::Const.World.Spawn.Troops.BanditLeader.Strength = 40;
-::Const.World.Spawn.Troops.BanditLeader.Cost = 35;
+::Const.World.Spawn.Troops.BanditLeader.Cost = 40;
+::Const.World.Spawn.Troops.HedgeKnight.Strength = 45;
```

Comment:
You set only the Strength, but not the Cost for HedgeKnight, Swordmaster, MasterArcher. Please set the Cost too.